### PR TITLE
invite friend modal added

### PR DESCRIPTION
**Pull request by isivak-fevo**

### CODE REVIEW

File: client/src/components/side-bar/components/inviteFriends.tsx [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+      addInviteEmail(user: $user, inviteEmail: $inviteEmail){
+        id
+        inviteEmail
+      }
+    }
+  `
+  const [addInviteEmail] = useMutation(ADD_INVITE_EMAIL)
+
+  return (
+    <>
```
Comment (cyberduck1) [inline client/src/components/side-bar/components/inviteFriends.tsx]: Why the empty tag?

**Comment by cyberduck1** [comment]
All looks good... 
I do have one question tho.
The invite component, look like it's addressing to invite a user to the userspace?

I thought that it will be available inside of a todo list.
And a user will be able to invite some other user to collab on the specific list....

File: client/src/components/side-bar/components/inviteFriends.tsx (hunk 1 above)
Comment (isivak-fevo) [reply client/src/components/side-bar/components/inviteFriends.tsx]: It is actually fragment, you can use it just like I used.
https://reactjs.org/docs/fragments.html#short-syntax

Comment (cyberduck1) [reply client/src/components/side-bar/components/inviteFriends.tsx]: OHHHH ok.... didn't know.....
Thanks for letting me know :)

**Pull request merged**